volume/mounts/mounts.go

```diff
@@ -62,6 +62,12 @@ type MountPoint struct {
 	// Sepc is a copy of the API request that created this mount.
 	Spec mounttypes.Mount
 
+	// Some bind mounts should not be automatically created.
+	// (Some are auto-created for backwards-compatability)
+	// This is checked on the API but setting this here prevents race conditions.
+	// where a bind dir existed during validation was removed before reaching the setup code.
+	DisableBindmountAutoCreate bool
```

Comment:
Maybe something simpler, like `SkipMountPointCreation` or `DoNotCreateMountPoint`?

Reply:
I mean, the name doesn't have to contain BindMount, also it's not clear from yours what is being auto created (a mount point).

Reply:
I like it, also makes it more straightforward to just skip setup earlier.

Reply:
Updated.